yaml_files/simphony_metadata.yml

```diff
@@ -438,9 +513,12 @@ CUDS_KEYS:
       scope: CUBA.USER
 
   BOX:
-    definition: A simple hexahedron box object
+    definition: A simple hexahedron (with six faces) simulation box defined by the three vectors and three directions. The condition should be specified for each direction (two faces at a time).   
```

Comment:
When defining 3x3 matrices representing three vectors, it's also useful to be clear if they are along the rows or the columns. I've seen so many errors due to this that it's never bad to overspecify.